util/weak_stub_maker.pl

```diff
@@ -2,10 +2,27 @@
 
 use warnings;
 use Text::ParseWords;
+use Getopt::Long;
 
+my $foundation = 0;
+GetOptions ('foundation|f' => \$foundation);
 my $inputFile = $ARGV[0];
 my $outputFile = $ARGV[1];
 
+my $load = '';
+my $unload = '';
+my $resolve = '';
+
+if ($foundation == 1) {
+    $load = "MCSDylibLoadModule";
+    $unload = "MCSDylibUnloadModule";
+    $resolve = "MCSDylibResolveModuleSymbol";
+} else {
+    $load = "MCU_loadmodule";
+    $unload = "MCU_unloadmodule";
+    $resolve = "MCU_resolvemodulesymbol";
+}
+
 # Read all of the input data
```

Comment:
Could we just change the weak stub maker to always use the libfoundation implementations instead of the MCU_* ones?